phase3/Kconfig

```diff
@@ -38,6 +38,13 @@ config phase3.weave_net
 	default n
 	help
 	  Should we deploy weave-net? You likely only want this if using kubeadm for phase2.
+
+config phase3.gce_storage_class
+	depends on phase2.enable_cloud_provider
+	bool "Deploy default storage class?"
```

Comment:
Is there a case where someone might want this enabled but not have the phase1 provider be gce? I'm wondering if this setting should also just depend on the phase1 provider option being set to gce.